experiments/large-result-sets/README.md

```diff
@@ -0,0 +1,61 @@
+# Performance gains for large result sets
+
+This experiment demonstrates the advantage of using [Query Builder's `p.guessTotal`](https://docs.adobe.com/content/help/en/experience-manager-65/developing/platform/query-builder/querybuilder-predicate-reference.html#root) parameter to return a subset of Query Builder query results.
+
+## Problem
+
+By default, Query Builder reads the entire result set to determine the total number of hits for a query which can be costly in terms of both execution time and memory usage for queries that have large result sets. In this experiment, we'll attempt to show the performance benefit of using the `p.guessTotal` parameter to avoid calculating the exact size of the full result set.
+
+## Setup
+
+AEM's QueryBuilder debugger tool will be used to execute [QueryBuilder-based](https://docs.adobe.com/help/en/experience-manager-65/developing/platform/query-builder/querybuilder-api.html) queries on the JCR (Java Content Repository).
+
+## Test #1: `Calculate the exact total`
+
+1. Navigate to AEM's QueryBuilder debugger tool at: http://localhost:4502/libs/cq/search/content/querydebug.html.
+
+2. Find all items in the /content folder in the JCR by typing the following text in the text area:
```

Comment:
I believe number lists just use 1. (i.e. do not increment).  For example:
1. First Point
1. Second Point
1. Third Point

See https://guides.github.com/features/mastering-markdown/#examples